documentation/modules/con-creating-managing-connectors.adoc

```diff
@@ -10,7 +10,7 @@ When you have created a container image for your connector plug-in, you need to
 
 * Check the status of a connector instance
 * Increase or decrease the number of tasks for a connector instance
-* Restart failed tasks
+* Restart failed tasks (not supported by `KafkaConnector` resource)
```

Comment:
Should we keep the information that this can be done through the REST API?

Reply:
We haven't mentioned 'KafkaConnector' resources yet, so I thought maybe to switch around like this:

![image](https://user-images.githubusercontent.com/47596553/72710199-75f24e00-3b5e-11ea-81e2-ca243de3ab52.png)

And then we probably don't need the note.

Reply:
Great suggestion. I'm happy for @sjwoodman to update the concept to match Paul's text above.

Reply:
@scholzj - "Should we keep the information that this can be done through the REST API?" - 

I think that is clear - let's try to avoid any extra words if we can!

Reply:
Thanks - I like Paul's suggestion so have updated the PR accordingly